Tests/DiligentToolsGPUTest/src/RenderStateNotationLoader/RenderStateNotationLoaderTest.cpp

```diff
@@ -83,6 +83,27 @@ TEST(Tools_RenderStateNotationLoader, BasicTest)
     RefCntAutoPtr<IPipelineState> pPSO;
     pLoader->LoadPipelineState(PipelineLI, &pPSO);
     ASSERT_NE(pPSO, nullptr);
+
+    PipelineStateDesc PipelineStateDescReference{};
+    PipelineStateDescReference.Name         = "GeometryOpaque";
+    PipelineStateDescReference.PipelineType = PIPELINE_TYPE_GRAPHICS;
+    ASSERT_EQ(PipelineStateDescReference, pPSO->GetDesc());
```

Comment:
This should be EXPECT_EQ as this is non-critical check and the test may continue if it fails (in contrast to previous ASSERT_NE(pPSO, nullptr), wher if pPSO is null, the test can't proceed)